charmverse/app.charmverse.io
# Guest invite

To Test:
1) share a page with a user by new email (eg matt.casey+12323@charmverse.io)
2) use the link in an incognito window. Hit "Submit" to receive a magic link
3) log in, using incognito link. Profit!

Notes:
- the email notice is sent every time a page is shared to a user now, instead of only the first time, so long as they have a verified email or google account w/that email. So this works for members that are not just guests
- if a user has access to the page already, they'll just be redirected to it
